testing/functional_tests/test_automate_training.py

```diff
@@ -10,7 +11,7 @@ def setup_function():
 
 
 @pytest.mark.script_launch_mode('subprocess')
-def test_automate_training(script_runner):
+def test_automate_training(get_functional_test_files, script_runner):
```

Comment:
By adding this Fixture reference, it signify that the unit test require files downloaded for it.